src/components/InForm/InForm.tsx

```diff
@@ -103,44 +113,54 @@ export const InForm: React.FunctionComponent<any> = (props) => {
   }
 
   return (
-    <div>
+    <>
       <Stack>
         <Stack.Item>
-          <TextField
-            label="Employee Name"
+          <Label htmlFor={empNameId}>Employee Name</Label>
+          <Input
+            id={empNameId}
             value={formData.empName}
             onChange={onEmpNameChange}
           />
-          <ChoiceGroup selectedKey={formData.empType} options={EMPTYPES} onChange={onEmpTypeChange} label="Employee Type" />
+          <Label htmlFor={empTypeId}>Employee Type</Label>
+          <RadioGroup value={formData.empType} onChange={onEmpTypeChange}>
+            {EMPTYPES.map((empType, i) => {
+              return (<Radio value={empType.value} label={empType.label} />)
+            })}
+          </RadioGroup>
+          <Label htmlFor={gradeRankId}>Grade/Rank</Label>
           <ComboBox
+            id={gradeRankId}
             selectedKey={formData.gradeRank}
-            label="Grade/Rank"
             autoComplete="on"
             options={gradeRankOptions}
             onChange={onGradeChange}
             dropdownWidth={100}
             disabled={formData.empType === "ctr"}
           />
-          <Toggle label="Local or Remote" onText="Remote" offText="Local" checked={formData.isRemote} onChange={onLocalRemote} />
+          <Label htmlFor={localRemoteId}>Local or Remote?</Label>
+          <Switch id={localRemoteId} label={formData.isRemote ? 'Remote' : 'Local'} checked={formData.isRemote} onChange={onLocalRemote} />
+          <Label htmlFor={arrivalDateId}>Select estimated arrival date</Label>
           <DatePicker
+            id={arrivalDateId}
             placeholder="Select estimated arrival date"
             ariaLabel="Select an estimated arrival date"
-            label="Select estimated arrival date"
           />
+          <Label htmlFor={officeId}>Office</Label>
           <ComboBox
+            id={officeId}
             selectedKey={formData.office}
-            label="Office"
             autoComplete="on"
             options={OFFICES}
             onChange={onOfficeChange}
             dropdownWidth={100}
           />
-          <Label>Supervisor/Government Lead</Label>
-          <PeoplePicker
-
+          <Label htmlFor={supervisorId}>Supervisor/Government Lead</Label>
+          <PeoplePicker id={supervisorId}
           />
-          {formData.empType === 'civ' ? <Toggle label="Is Employee a New to Air Force Civilian?" onText="Yes" offText="No" onChange={onNewCiv} /> : null}
-          <PrimaryButton text="Create In Processing Record" onClick={reviewRecord}></PrimaryButton>
+
+          {formData.empType === 'civ' ? <><Label htmlFor={newCivId}>Is Employee a New to Air Force Civilian?</Label><Switch id={newCivId} label={formData.isNewCiv ? 'Yes' : 'No'} onChange={onNewCiv} /></> : null}
+          <Button appearance="primary" onClick={reviewRecord}>Create In Processing Record</Button>
```

Comment:
In your JSX, rather than using a ternary operator with a null return, use the && operator
```
// Avoid
{conditionCheck ? <MyComponent /> : null}
// Instead use
{conditionCheck && <MyComponent />}
```

Button does not appear on it's own line if first line returns a component

Reply:
Good call.  Fixed it.